components/portal/main/controllers.js

```diff
@@ -211,10 +211,19 @@ define(['angular', 'require'], function(angular, require) {
         vm.campusId = vm.user[vm.campusIdAttribute];
       }
 
-      $document[0].dispatchEvent(new CustomEvent('myuw-login', {
-        bubbles: true,
-        detail: {person: {'firstName': vm.username},
-      }}));
+     // Check session info to identify if user is a guest
+      if (vm.user.firstName && vm.user.lastName) {
+        $document[0].dispatchEvent(new CustomEvent('myuw-login', {
+          bubbles: true,
+          detail: {person: {'firstName': vm.username},
+        }}));
+      } else {
+       $document[0].dispatchEvent(new CustomEvent('myuw-login', {
+          bubbles: false,
```

Comment:
Why does an event with a null person not-bubble but an event with a person bubble?